README.md

```diff
@@ -71,14 +71,47 @@ terraform plan
 terraform apply
 ```
 
-### NFS server licensing
 
-User network home directories are provided by [SoftNAS from AWS Marketplace](https://aws.amazon.com/marketplace/pp/B01BJC4JI6?qid=1495795249740&sr=0-3&ref_=srh_res_product_title). The default image defined in Terraform uses a per-terabyte consumption billing model; if required storage exceeds 1TB the [per server version](https://aws.amazon.com/marketplace/pp/B00PJ9FGVU?qid=1495795249740&sr=0-2&ref_=srh_res_product_title) will be more cost effective.
-
-**Before proceeding with setup of a new environment the SoftNAS subscription must be accepted manually at the URL(s) above.**
+## Environment setup
 
 ### Defining new environment
 
+#### SoftNAS NFS server setup
+
+User network home directories are provided by SoftNAS from AWS Marketplace. There are a few different versions e.g.:
+
+* [SoftNAS from AWS Marketplace](https://aws.amazon.com/marketplace/pp/B01BJC4JI6?qid=1495795249740&sr=0-3&ref_=srh_res_product_title) is "For Lower Compute Requirements".
+* [SoftNAS Cloud Developer Edition 4.0.x](https://aws.amazon.com/marketplace/pp/B06Y5W7TKY?qid=1533814033150&sr=0-4&ref_=srh_res_product_title) is limited to 250GB but the software is **free** - you just pay $0.085/hr for c5.large EC2 machine.
+
+There are about 20 SoftNAS options on AWS Marketplace, with varying cost models etc, so it's worth evaluating which ones suit your purpose.
+
+Once selected, on the SoftNAS product web page you need to:
+
+1. Click "Continue to Subscribe"
+2. Click "Accept terms"
+3. Wait 30 seconds before the flash message appears "Thank you for subscribing to this product!"
+4. Click "Continue to Configuration" (which has also appeared)
+5. Configure:
+
+   * Region: choose the same as chosen for the rest of your platform (e.g. EU Ireland)
+
+6. Click "Continue to Launch"
+
+   * EC2 Instance Type - select a suitable one, considering cost. Record the instance type (e.g. `m5.large`) - you'll use this in your .tfvars file in a moment.
+   * Key pair - create one called "softnas-$ENVNAME" (replacing the $ENVNAME) and save the private key (.pem file) locally
```

Comment:
Do we need to do something special about the softnas key pair? This is not kept by AWS and we'll need it to access softness right?

Reply:
I think this is the case David so it may be the case of adding a note in the readme in case someone throw these keypair file away thinking it can be retrieved somehow.

Reply:
I've added a note about the key here: https://github.com/ministryofjustice/analytics-platform-ops/pull/191/files#diff-04c6e90faac2675aa89e2176d2eec7d8R186